### PR TITLE
Serve swagger docs

### PR DESCRIPTION
**Pull request by anilCSE**
- Rename `querier.proto` to `query.proto`
- Integrate grpc-gateway endpoint `/swagger/` in app.go

### CODE REVIEW

**Review by ssandeep — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,6 @@ logs
 .idea
 .vscode
 .cache
+
+# swagger-gen
+/tmp-swagger-gen
```
Comment (jdkanani) [inline .gitignore:18]: Please add a new line at end of the file

File: client/docs/config.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            "url": "./client/docs/cosmos/params/v1beta1/query.swagger.json"
+        },
+        {
+            "url": "./client/docs/cosmos-legacy-swagger.yaml",
+            "dereference": {
+                "circular": "ignore"
+            }
+        }
+    ]
+}
```
Comment (jdkanani) [inline client/docs/config.json:43]: Same here. Please new line at end of the file.

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: client/docs/cosmos-legacy-swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,2598 @@
+---
```
Comment (jdkanani) [inline client/docs/cosmos-legacy-swagger.yaml:1]: Do we need legacy file?

Comment (anilCSE) [reply client/docs/cosmos-legacy-swagger.yaml:1]: There are a few tendermint RPC endpoints, for which we don't have gRPC alertnatives. This legacy swagger has those endpoints. May be we can remove some APIs from there, will update

Comment (jdkanani) [reply client/docs/cosmos-legacy-swagger.yaml:1]: Got it.

**Review by jdkanani — APPROVED** [review]

**Pull request merged**